third_party/move/move-compiler-v2/tests/file-format-generator/vector.exp

```diff
@@ -80,6 +80,8 @@ L1:	loc1: u64
 L2:	loc2: u64
 L3:	loc3: u64
 L4:	loc4: u64
+L5:	loc5: u64
```

Comment:
Note that this is the unoptimized output (and therefore, it is worse due the change in this PR).

The optimized output is better than this, and does not change due to the fix.